interface/CommonPlot.h

```diff
@@ -161,6 +161,8 @@ class CommonPlot {
         graph->GetYaxis()->SetLabelOffset(p_desc.axis_label_offset.y());
         graph->GetXaxis()->SetLimits(p_desc.x_range.min(), p_desc.x_range.max());
         graph->GetYaxis()->SetRangeUser(p_desc.y_range.min(), p_desc.y_range.max());
+        graph->GetXaxis()->SetNoExponent(kTRUE);
+        graph->GetXaxis()->SetMoreLogLabels(kTRUE);
```

Comment:
In the future, will need to move these parameters into config. But should be ok for now, since general review of all HH plotting tools is needed.